test/connectors/httpParsers/VatReturnHttpParserSpec.scala

```diff
@@ -44,6 +44,44 @@ class VatReturnHttpParserSpec extends UnitSpec {
         "totalAcquisitionsExVAT" -> 545645
       )))
 
+
```

Comment:
Unused import in this file, as well as a spelling mistake further down below from an earlier commit.